### ISSUE TITLE
Discover by organization cannot find any dependencies

### ISSUE THREAD

**Issue by coni2k** [opencollective/backyourstack#53]
When I try to search dependencies by entering organization name, it cannot find any dependencies:
Link: https://backyourstack.com/forCrowd
Result: 10 repositories depending on 0 Open Source projects.

But when I search directly by using "package.json" file that is under one of the repositories of my organization, it finds two dependencies that have Open Collective account ("html-webpack-plugin", "webpack"):

Repo: https://github.com/forCrowd/Backbone
Package.json file: https://github.com/forCrowd/Backbone/blob/master/AngularClient/package.json

**Comment by znarf** [comment]
Sounds similar to https://github.com/opencollective/backyourstack/issues/51

The issue is that the `package.json` file is in the 'AngularClient' folder and not at root level.

It's complicated to solve because we don't download the whole repository.

That's said, the dependencies are properly detected by GitHub https://github.com/forCrowd/Backbone/network/dependencies, so maybe we should contact them and collaborate on that.

**Comment by coni2k** [comment]
Yep, it looks like it's the same issue, and you're right, somehow you have to be able to see/list all files from all repositories, so you can have better results.

Looks like a brilliant initiative on top of Open Collective and I'm sure you will improve it in time. Just wanted to report what I came across. Please let me know if there is anything else I can do.
